src/main/java/com/uber/cadence/worker/WorkerOptions.java

```diff
@@ -51,28 +49,6 @@
     private Scope metricsScope;
     private boolean enableLoggingInReplay;
 
-    /**
```

Comment:
Just a reminder we may need to mention these breaking changes in next release note.

Reply:
Yes. The next release will be 3.0